src/NEO/Signer.h

```diff
@@ -0,0 +1,34 @@
+// Copyright © 2017-2019 Trust Wallet.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+#pragma once
+
+#include "Address.h"
+#include "Transaction.h"
+#include "../PrivateKey.h"
+#include "../Data.h"
+
+namespace TW::NEO {
+    class Signer {
+    private:
+        Data publicKey;
+        TW::PrivateKey privateKey;
+        std::string address;
+    public:
+        explicit Signer(const TW::PrivateKey &priKey);
+        PrivateKey getPrivateKey() const;
```

Comment:
`getPrivateKey`, `getPublicKey` and `getAddress` are quite trivial, can we remove it from signer?